### PR TITLE
Material ui is a go

### PR DESCRIPTION
**Pull request by pachown**

### CODE REVIEW

**Comment by pachown** [comment]
[Added Material UI - basic component is finished!](https://trello.com/c/m8L6olmp/74-added-material-ui-basic-component-is-finished)

File: Client/src/components/Restaurants.jsx [hunk 1]
```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
 import Restaurant from './Restaurant';
 
 const Restaurants = ({ restaurants }) => (
   <div>
```
Comment (this-Cunningham) [inline Client/src/components/Restaurants.jsx:7]: nice update here adding dynamic elements

File: Database/attraction.js [hunk 2]
```diff
@@ -39,12 +39,7 @@ module.exports = {
   create: (req, res) => {
     const obj = {
       name: req.body.name,
-
-      ratings: {
-        avg: req.body.ratings.avg,
-        total: req.body.ratings.total,
-      },
-
+      ratings: req.body.ratings,
```
Comment (this-Cunningham) [inline Database/attraction.js:42]: nice simplification here

File: Database/location.js [hunk 3]
```diff
@@ -77,21 +66,10 @@ module.exports = {
...
-      coords: {
-        lat: req.body.coords.lat,
-        long: req.body.coords.long,
-      },
-      ratings: {
-        avg: req.body.ratings.avg,
-        total: req.body.ratings.total,
-      },
+      coords: req.body.coords,
+      ratings: req.body.ratings,
```
Comment (this-Cunningham) [inline Database/location.js:70]: much cleaner here, nice

**Review by this-Cunningham — APPROVED** [review]
looks like great changes!

**Pull request merged**